lib/sprockets/sass_processor.rb

```diff
@@ -68,9 +69,9 @@ def call(input)
           environment: input[:environment],
           dependencies: context.metadata[:dependencies]
         }
-      }
+      }.merge!(@sass_config)
```

Comment:
Shouldn't we do some sort of deep merge here? The `load_paths` entry gets replaced by an empty `Array` from the defaults of [`sass-rails`](https://github.com/rails/sass-rails/blob/a11d4c7989a2b3e82b780e4c363c7e5d28c4b739/lib/sass/rails/railtie.rb#L19-L20) and it seems to be breaking `@import` statements.